pelican-gfm/Readme.md

```diff
@@ -0,0 +1,22 @@
+pelican-gfm
+===========
+A reader that leverages GitHub's C-based markdown library to translate GitHub Flavored Markdown to html.
+
+Requirements
+============
+
+pelican-gfm has no requirements that are outside of the python standard library aside from pelican itself.
```

Comment:
Out of curiosity, have you tried https://pypi.org/project/gfm/ or https://pypi.org/project/py-gfm/ ?
Do they provide what you need maybe, already bundled as Python packages ?

Reply:
I believe that there were pieces of the github markup that those packages did not readily provide. I'm not 100% sure as I Inherited the base of this.